subprojects/logging/src/main/java/org/gradle/internal/logging/sink/OutputEventRenderer.java

```diff
@@ -79,17 +80,20 @@ public OutputEventRenderer(final Clock clock) {
         OutputEventListener stdOutChain = new LazyListener(new Factory<OutputEventListener>() {
             @Override
             public OutputEventListener create() {
-                return onNonError(new UserInputStandardOutputRenderer(new BuildLogLevelFilterRenderer(new ProgressLogEventGenerator(new StyledTextOutputBackedRenderer(new StreamingStyledTextOutput(stdoutListeners.getSource())), false)), clock));
+                return new UserInputStandardOutputRenderer(new StyledTextOutputBackedRenderer(new StreamingStyledTextOutput(stdoutListeners.getSource())), clock);
             }
         });
-        formatters.add(stdOutChain);
         OutputEventListener stdErrChain = new LazyListener(new Factory<OutputEventListener>() {
             @Override
             public OutputEventListener create() {
-                return onError(new BuildLogLevelFilterRenderer(new ProgressLogEventGenerator(new StyledTextOutputBackedRenderer(new StreamingStyledTextOutput(stderrListeners.getSource())), false)));
+                return new StyledTextOutputBackedRenderer(new StreamingStyledTextOutput(stderrListeners.getSource()));
             }
         });
-        formatters.add(stdErrChain);
+        formatters.add(
+            new BuildLogLevelFilterRenderer(
```

Comment:
I wonder if we shouldn't move the lazy wrapper that is around the stdout chain and the stderr chain to here.

Reply:
Yeah, that's a good point.  I've made that change, too.